### DISCUSSION TITLE
I think cypress substitution of iframe to placeholder is breaking my test

### DISCUSSION THREAD

**Discussion by ronaldohoch** [cypress-io/cypress#17979] [Questions and Help]
Hello Cypress team!
I have a project that uses iframe on it. This project is a webpage builder based on Grapes JS, here is the demo page of it: https://grapesjs.com/demo.html

The blocks on the right column are the components that we use to build the page and the big blank column at left is where iframe stands. So, we drag a block from outside to inside the iframe, this way: 
![image](https://user-images.githubusercontent.com/2287371/132064543-fa2a450d-1703-4335-a149-4abb3f56c0c5.png)

But when we try to use Cypress to test it, the iframe on it has a strange behavior where it doesn't allow to drop the block.
I have a simple test so you can see it...

I'm using the cypress-drag-drop plugin: https://github.com/4teamwork/cypress-drag-drop
This is the code:

```
describe('My test', () => {
    it('Visit', () => {
        cy.visit('https://grapesjs.com/demo.html');
    })
    it("Should drop",()=>{
        cy.get('.gjs-f-b3').drag('.gjs-frame',{position:"center"});
    });
});
```

The print of test

Selecting the block.:
![image](https://user-images.githubusercontent.com/2287371/132064638-68601fc1-1c4a-4fd9-9217-f86003df3b2f.png)

Drop it to center of iframe.:
![image](https://user-images.githubusercontent.com/2287371/132064696-f3486548-538f-4be1-aa2f-483377d098f6.png)

So, i think this placeholder is breaking the test :/

**Comment by avgulavani** [comment]
Hi @ronaldohoch I have been trying to do the same thing for many days but in vain. 
I need to update the drag-and-drop tool based on Grapes JS but things ate not working in the correct direction.
Could you please help me here? Appreciate any leads on this topic.

![image](https://github.com/user-attachments/assets/3037b24b-fef5-4640-835e-c2d24667cf85)

**Reply by ronaldohoch** [reply]
Hi, i couldn't make e2e testing on grapesjs :/
